pose_est_nets/models/heatmap_tracker.py

```diff
@@ -214,7 +227,7 @@ def heatmaps_from_representations(
     @typechecked
     def forward(
         self, images: TensorType["batch", "channels":3, "image_height", "image_width"]
```

Comment:
if we're adding `float`s (which I hope won't hurt when we try half precision), I'd suggest adding one for `images`.